mobile/examples/cc/fetch_client/BUILD

```diff
@@ -43,27 +43,3 @@ cc_binary(
         "@envoy//source/exe:process_wide_lib",
     ],
 )
-
-cc_test(
-    name = "fetch_client_test",
-    srcs = ["fetch_client_test.cc"],
```

Comment:
Do you still need the .cc file?

Reply:
There is no .cc such file in the repo, so I think there's nothing to remove.